.github/workflows/windows.yml

```diff
@@ -128,11 +125,11 @@ jobs:
       fail-fast: false
       matrix:
         combo:
```

Comment:
We benefit from `latest`, consistently, and thus this should break much less in the future.